client/my-sites/site-settings/administration/index.jsx

```diff
@@ -0,0 +1,22 @@
+import { translate } from 'i18n-calypso';
```

Comment:
So the idea is to put the components in `client/sites/settings/*` instead of in `client/my-sites`. The latter is the location for the nav-unified Calypso pages, which we want to move away.

Can we put this as `client/sites/settings/administration/index.jsx`? (Similar to the other new files in this PR)